### PR TITLE
Update accepted file types in Indirect Corrections documentaton

### LINKED ISSUE #33429: Update documentation for Indirect corrections to refelect acceptable workspaces.
**Original reporter:** @SpencerHowells

The documentation for Indirect Corrections says they accept _sqw files as inputs, this is not correct, it should be corrected.

### PR DESCRIPTION
**Pull request by Harrietbrown**
**Description of work.**
This PR updates the Indirect corrections documentation list of allowed files or workspaces.

**Report to:** @SpencerHowells

**To test:**

Check documentation

Fixes #33429

*This does not require release notes* because It is a simple change to documentation

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/indirect/Indirect Corrections.rst [hunk 1]
```diff
@@ -55,10 +55,10 @@ Options
 ~~~~~~~
 
 Sample
-  Either a reduced file (_red.nxs) or workspace (_red) or an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) that represents the sample.
+  Either a reduced file (_red.nxs) or workspace (_red), an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) or an ELF file (_elf.nxs) or workspace (_elf) that represents the sample.
```
Comment (ConorMFinn) [inline docs/source/interfaces/indirect/Indirect Corrections.rst:58]: Are the `\omega` sections meant to be appearing as symbols?

May need to be:
```
    :math:`S(Q,\omega)`
```

File: docs/source/interfaces/indirect/Indirect Corrections.rst [hunk 2]
```diff
@@ -55,10 +55,10 @@ Options
...
 
 Container
-  Either a reduced file (_red.nxs) or workspace (_red) or an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) that represents the container.
+  Either a reduced file (_red.nxs) or workspace (_red), an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) or an ELF file (_elf.nxs) or workspace (_elf) that represents the container.
```
Comment (ConorMFinn) [inline docs/source/interfaces/indirect/Indirect Corrections.rst:61]: Super nitpicky, I know, but could we have another comma here to separate out the list?
```suggestion
  Either a reduced file (_red.nxs) or workspace (_red), an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw), or an ELF file (_elf.nxs) or workspace (_elf) that represents the container.
```

**Review by cailafinn — CHANGES_REQUESTED** [review]
Nice, just a couple of tiny nitpicks

**Review by cailafinn — APPROVED** [review]
Looks good to me now 👍

**Review by DavidFair — APPROVED** [review]

**Pull request merged**